evaluations/README.md

```diff
@@ -6,6 +6,7 @@ scripts are available, with reference to corresponding figure in our white paper
  - `pose_graph_synthetic.sh`: Same as above, but can change linear solver and problem size (Fig. 2).
  - `pose_graph_cube.sh`: Same as above, but using the cube data for Ceres comparison (Fig. 3). 
  - `backward_modes_tactile.sh`: Runs tactile state estimation with different backward modes (Fig. 4).
+ - `autodiff_cost_function_ablation.sh`: Runs homography estimation examples with different autograd modes.
```

Comment:
Since the profiling script is moved to evaluations we should update both readme to reflect that.